src/config-factory.test.ts

```
@@ -333,3 +338,113 @@ describe("config-factory", () => {
     });
   });
 });
+
+describe("config change event emitter", () => {
+  // Keep track of all unsubscribe functions so we never leak listeners
+  // between tests.
+  const unsubscribers: Array<() => void> = [];
+
+  afterEach(() => {
+    unsubscribers.forEach((fn) => fn());
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)